style.css

```diff
@@ -0,0 +1,92 @@
+html {
+  font-size: 10px;
+}
+body {
+  font-size: 1.6rem;
+}
+.container {
+  max-width: 120rem;
+  margin: 0 auto;
+}
+.todo-list-container {
+  margin: 0 auto;
+  width: 50%;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+.todo-list-item {
+  margin: 2rem auto;
+  height: 5rem;
+  width: 50%;
+  display: flex;
+  flex-direction: row;
+}
+.todo-list-item__text {
+  display: flex;
+  align-items: center;
+  border-bottom: 0.1rem solid #afafaf;
+  width: 25rem;
+  margin-left: 5rem;
+  text-transform: capitalize;
+}
+.todo-list-item__input {
+  display: flex;
+  align-items: center;
+}
+.todo-list-item__input input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+}
+
+.todo-list-item__input .todo__check-custom {
+  position: absolute;
+  height: 2.4rem;
+  width: 2.4rem;
+  background-color: transparent;
+  border-radius: 0.5rem;
+  border: 0.1rem solid #8b8b8b;
+  cursor: pointer;
+}
+
+.todo-list-item__input .todo__check-custom::after {
+  position: absolute;
+  content: "";
+  left: 1.2rem;
+  top: 1.2rem;
+  height: 0;
+  width: 0;
+  border-radius: 0.5rem;
+  border: solid #8b8b8b;
+  border-width: 0 0.1rem 0.1rem 0;
+  -webkit-transform: rotate(0deg) scale(0);
+  -ms-transform: rotate(0deg) scale(0);
+  transform: rotate(0deg) scale(0);
+  opacity: 1;
+}
+
+.todo-list-item__input input:checked ~ .todo__check-custom {
+  background-color: #fd3d6c;
+  border-radius: 0.5rem;
+  -webkit-transform: rotate(0deg) scale(1);
+  -ms-transform: rotate(0deg) scale(1);
+  transform: rotate(0deg) scale(1);
+  opacity: 1;
+  border: 2px solid transparent;
+}
+
+.todo-list-item__input input:checked ~ .todo__check-custom::after {
+  -webkit-transform: rotate(45deg) scale(1);
+  -ms-transform: rotate(45deg) scale(1);
+  transform: rotate(45deg) scale(1);
+  opacity: 1;
+  left: 0.8rem;
+  top: 0.3rem;
+  width: 0.6rem;
+  height: 1.2rem;
+  border: solid #ffffff;
+  border-width: 00 0.1rem 0.1rem 0;
+  background-color: transparent;
+  border-radius: 0;
```

Comment:
Unnecessary

Reply:
Is necessary. Border radius is set to 0.5rem on line 69 this converts the tick back to standard border radius